runtime/oti/j9nonbuilder.h

```diff
@@ -4058,6 +4058,25 @@ typedef struct J9JITConfig {
 #endif /* J9VM_OPT_JITSERVER */
 } J9JITConfig;
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+typedef BOOLEAN (*criuHookFunc)(struct J9JavaVM *vm, void *userData, j9object_t obj, BOOLEAN isRestoreHook);
```

Comment:
we dont need `BOOLEAN isRestoreHook` since the record already has that info

Reply:
Removed.